plugins/destination/postgresql/client/types.go

```diff
@@ -94,3 +94,86 @@ func (*Client) SchemaTypeToCockroach(t schema.ValueType) string {
 		panic("unknown type " + t.String())
 	}
 }
+
+func (c *Client) PgToSchemaType(t string) schema.ValueType {
+	switch c.pgType {
+	case pgTypeCockroachDB:
+		return c.CockroachToSchemaType(t)
+	default:
+		return c.Pg10ToSchemaType(t)
+	}
+}
+
+func (*Client) Pg10ToSchemaType(t string) schema.ValueType {
+	switch t {
+	case "boolean":
+		return schema.TypeBool
+	case "bigint", "integer":
```

Comment:
Interesting, I guess there is some information lost in the conversion here because our types don't map 1:1 to postgres 🤔 So if you have a source table and write it back to postgres, the schemas won't match exactly (integers will become bigints). In this case it's fine, but hopefully we don't run into irreconcilable cases in other databases...

Reply:
Good catch. I guess we need to transform the incoming tables to pgtables so we can compare them?

Reply:
Yeah I think in this case it's fine until we add BigInt type to our type system but if the information loss is the otherway around then we will need to transform first the Table -> DbTable and only then compare what we got from database table.